MultiPartyExample/MainViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  MainViewController.swift
+//  MultiPartyExample
+//
+//  Copyright © 2019 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+class MainViewController: UIViewController {
+
+    // MARK: View Controller Members
+
+    // Configure access token manually for testing, if desired! Create one manually in the console
+    // at https://www.twilio.com/console/video/runtime/testing-tools
+    var accessToken = "TWILIO_ACCESS_TOKEN"
+
+    // Configure remote URL to fetch token from
+    var tokenUrl = "http://localhost:8000/token.php"
+
+    // Maximum bitrate (in kbps) used to send video.
+    static let kMaxVideoBitrate = UInt(1500)
+
+    // MARK: UI Element Outlets and handles
+    @IBOutlet weak var connectButton: UIButton?
```

Comment:
Typically IBOutlets are force unwrapped and not optionals because the storyboard is ensuring that they are present when the view controller is initialized.